boot/assets.go

```diff
@@ -847,3 +847,53 @@ func observeSuccessfulBootAssets(m *Modeenv) (newM *Modeenv, drop []*trackedAsse
 	}
 	return newM, drop, nil
 }
+
+func observeSuccessfulCommandLine(m *Modeenv) (*Modeenv, error) {
```

Comment:
should these functions live in cmdline.go instead perhaps?

Reply:
and moved